docs/running-tests.md

```diff
@@ -6,38 +6,187 @@ In development, you can run tests by using Gulp.
 gulp test
 ```
 
-This will run the tests using PhantomJS and automatically watch for any file changes to run the tests again.
+This is a wrapper for the Jest CLI with the following options:
+
+  * watch - Enters Jest watch interactive CLI
+  * onlyChanged - Looks at your git integration for files that have changed and immediately runs the appropriate spec files
+
+The wrapper uses the Carbon Factory jest.conf.json file as a base and merges in any changes passed to the gulp task.
+
+e.g.
+
+The following gulp task will ingnore all __spec__.js and definition.js files in the src directory and turn off watch mode
+
+```js
+gulp.task('test', SpecTask({
+  jestConfig: {
+    path: '/src/***/**/!(__spec__|definition).js',
+    watch: false
+  }
+}));
+```
+
+The tests will run with coverage by default as defined by the base jest.config.json file in Carbon Factory.
+
+To open the coverage report can be opened from the top of your ui folder using the following command.
+
+```bash
+  open coverage/index.html
+```
+
+If we you want to define your own jest.conf.json file you can override the preset value by passing it to your gulp task.
```

Comment:
The order of topics seems to jump around a bit in this file. We've gone from overriding `jest.conf.json` values to opening code coverage reports and back to overriding `jest.conf.json` values. The next section jumps after here mentions code coverage again.

Also, is this bit here duplicating the earlier example where we override `jest.conf.json` values?